src/core/renderers/webgl/utils/RenderTarget.js

```diff
@@ -138,7 +138,7 @@ export default class RenderTarget
          *
          * @member {boolean}
```

Comment:
Is it possible to add `@default false` to the `root` doc? Not critical, but helpful to be more explicit.

Reply:
Its already `false` in constructor docs, but i added it anyway.